enterprise/server/auth/auth.go

```diff
@@ -472,19 +479,46 @@ func lookupUserFromSubID(env environment.Env, ctx context.Context, subID string)
 		if err := userRow.Take(user).Error; err != nil {
 			return err
 		}
-		groupRows, err := tx.Raw(`SELECT g.* FROM `+"`Groups`"+` as g JOIN UserGroups as ug
-                                          ON g.group_id = ug.group_group_id
-                                          WHERE ug.user_user_id = ?`, user.UserID).Rows()
+		groupRows, err := tx.Raw(`
+			SELECT
+			  g.user_id,
```

Comment:
nit: fix indent

Reply:
This indent is intentional since the field list is syntactically a "child" of the SELECT

Reply:
The indentation you had is good. I was pointing out that for some reason the indentation for the first field was different from the remaining fields.

Reply:
Gotcha, I think it was indented with spaces instead of tabs so I didn't notice it locally. Done